pkg/gateway/gateway.proto

```diff
@@ -365,3 +372,47 @@ message DeleteMachineResponse {
   bool ok = 1;
   string err_msg = 2;
 }
+
+message Token {
+  uint32 id = 1;
+  string external_id = 2;
```

Comment:
`external_id` should be `id` or `token_id`. The `id` (regular int, not uuid) is an internal id that we don't typically expose externally.